cli/plasmo/src/features/extension-devtools/scaffolds.ts

```diff
@@ -24,7 +24,7 @@ const generateScaffold = async (
   }
 
   const finalScaffold = Object.keys(replaceMap).reduce(
-    (html, key) => html.replaceAll(key, replaceMap[key]),
+    (html, key) => html.split(key).join(replaceMap[key]),
```

Comment:
This is very slow. Another solution is to use a global regex, but since Plasmo is made to target node16 and above, closing this as the OP did would be what I would suggest as well 👍 Great effort regardless :)